internal/endorser/endorser.go

```diff
@@ -93,9 +96,11 @@ func loadAndVerifyProvenances(provenanceURIs []string, binaryDigest string) (*am
 // binaryDigest as the subject. Provenances are consistent if they all have the same binary name and
 // binary digest. An error is returned if any of the conditions above are violated.
 // TODO(b/222440937): Document any additional checks.
-func verifyProvenances(provenances []slsa.ValidatedProvenance, binaryDigest string) error {
+func verifyProvenances(provenances []slsa.ValidatedProvenance, expected verifier.ProvenanceIR) error {
```

Comment:
```suggestion
func verifyProvenances(provenances []slsa.ValidatedProvenance, referenceValues verifier.ProvenanceIR) error {
```

Optional suggestion to be more specific :)